public/js/chat.js

```diff
@@ -38,6 +41,27 @@ var Chat = function(id, player) {
 		});
 	});
 
+	//pulling the emote json file from the dgg cdn, not sure how to do this without a cors proxy :(
+	$.get("https://cors-anywhere.herokuapp.com/https://cdn.destiny.gg/2.13.0/emotes/emotes.json", function(emoteList){
+		this.emoteList = emoteList;
+		//stolen from ceneza Blesstiny
+		self.emoteMap = new Map();
+		emoteList.forEach(v => self.emoteMap.set(v.prefix, v));
+	});
+
+	//stolen from ceneza Blesstiny
+	this.loadCss = function(url) {
```

Comment:
This could just be an actual `<link>` element in `/public/index.html` right? I don't think this url is being set dynamically as far as I can tell